frontend/src/components/TopNavBarRevamp.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { ReactElement } from 'react'
+import { useHistory } from 'react-router-dom'
+import { NavBar, Icon } from 'antd-mobile'
+import styled from 'styled-components'
+
+import 'antd-mobile/dist/antd-mobile.css'
+
+const TitleText = styled.text`
```

Comment:
consider using `h1` or any of the headings instead since `text` only exists in `svg` element

Reply:
Route is added as well.
Resolved in commit 8d8bcd18261ca08a4f4b82800eadadc527fb43b1.